docs/data/charts/tooltip/tooltip.md

```diff
@@ -97,6 +97,25 @@ It removes the header showing the x-axis value from the tooltip.
 To hide a series, the formatted value should be `null`.
 To display the series with a blank space, return an empty string.
 
+## Position
+
+By default the tooltip is placed relatively to the pointer position.
+If the pointer is not available, the item tooltip is placed relatively to the item.
+
+This behavior can be modified with the `anchor`, `position`, and `placement` props.
+
+- The `anchor: 'pointer' | 'node'` indicates if the tooltip should be placed relatively to the pointer or the node.
+- The `position: 'top' | 'right' | 'bottom' | 'left'` precise the anchor position compared to the node.
+- The `placement` is the tooltip placement from [PopperJS](https://popper.js.org/docs/v1/#popperplacements--codeenumcode). By default the same value as `position` if defined.
```

Comment:
It feels like we don't need both `position` and `placement`.

Reply:
> But its seems to be usefull. The `position` is than anchor placement and `placement` how does the tooltip place itself relatively to the anchor

It doesn't seem like this is how it works. It feels like we could just kill position in favour of placement.


https://github.com/user-attachments/assets/c8206e79-85d4-4db0-b8dd-0693a5fab9f3